habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -782,12 +780,14 @@ class HabitatSimRGBSensorConfig(SimulatorCameraSensorConfig):
 
 
 @dataclass
-class HabitatSimDepthSensorConfig(SimulatorDepthSensorConfig):
+class HabitatSimDepthSensorConfig(
```

Comment:
It's a broader refactor question if we want different dataclasses for each sensor type or if the config should be more general. Regardless, that is orthogonal to this PR just to keep in mind.

Reply:
Yes, we should have this discussion. Having a dataclass for each sensor allows to have type checking for each sensor, which actually helped us catch a few issues.